Command/MigrateCommand.php

```diff
@@ -100,7 +78,7 @@ protected function configure()
                 'force',
                 null,
                 InputOption::VALUE_NONE,
-                'Allow to import from the default environment'
+                'Allow to import from the default environment and/or to cancel draft revision'
```

Comment:
Why combine these two functions in 1 option? are we never going to import in the default environment, but without cancelling draft revisions?

Reply:
I had the same discussion with @Davidmattei we agree on this. The force should never be used (in theory). There is the recompute or the rebuild command regarding the default environment. You can also combine a rebuild and import from the previous index. If you import from the default environment you are saying that that the DB is not ok and should be aligned from the index.

Reply:
edb613bfbca9091fc7e285c8f2271b2d873c1068